docs/shared/recipes/running-tasks/customizing-inputs.md

```diff
@@ -1,13 +1,17 @@
 ---
 title: Fine-tuning Caching with Inputs
```

Comment:
This document isn't a recipe any more - it was border line before and has clearly crossed the line.  It's kind of hybrid between a reference doc and a concept page, but we should move it into the concept section if we're not going to re-write it completely as separate docs.